### ISSUE TITLE
About Code Style

### ISSUE THREAD

**Issue by ricky9090** [victorr/jsqueak#3]
Hi Victor

It seems that both the Eclipse and Intellij IDEA use a more modern Java code style by default.
More specifically：
[Google Java Style Guide](https://google.github.io/styleguide/javaguide.html)
[Firefox Java Code Style](https://firefox-source-docs.mozilla.org/code-quality/coding-style/coding_style_java.html)

The major difference :
Opening braces on same line, closing braces on a new line

I think we should rely on a more standard style, and it will be much more easier to maintain the project 😀

**Comment by victorr** [comment]
Hi Ricky,

I will be happy to continue accepting all PRs you send, but as you can see form the commit history, I did very little work on the project, and that was over 13 years ago. I think you shouldn't worry about me and work on your fork instead :-)

Kind regards.

**Comment by ricky9090** [comment]
> Hi Ricky,
> 
> I will be happy to continue accepting all PRs you send, but as you can see form the commit history, I did very little work on the project, and that was over 13 years ago. I think you shouldn't worry about me and work on your fork instead :-)
> 
> Kind regards.

Hi Victor,

I'm excited that you accepted my PR.
It's truly a long time since your last commit many years ago.

In fact, I just studied Smalltalk for a few months, and I found that JSqueak 
seems to be the perfect choice to study the implementation of an OOP 
language and its Virtual Machine.

Currently, I'm trying to improve the usability of JSqueak, fix bugs, and finish 
other primitives implementation.

Here is the branch I'm working on:
https://github.com/ricky9090/jsqueak/tree/study

With your permission, I will contine work on my fork based on your work 
(FileSystemPrimitives, Keyboard fixes), and open PR to your repo for important 
changes/improvements in the future.
